handler/game_handler.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../model/game'
+require_relative '../storage'
+
+class GameHandler
+  @storage = Storage.new
+  def self.add_game(app)
+    puts 'Please, create your game : '
+    puts 'multiplayer : '
+    multiplayer = gets.chomp
+    puts 'last_played_date : '
+    last_played_date = Date.parse(gets.chomp)
+    puts 'publish_date : '
+    publish_date = gets.chomp
+    game = Game.new(multiplayer, last_played_date, publish_date)
+    app.create_game(game)
+    puts 'game created successfully !!!'
+    @storage.save_game(app)
+  end
```

Comment:
- [ ] Good job adding a method to create the game, Kindly see error in the screenshot below after a game is created, please fix it 👍🏿  
![Screenshot_548](https://user-images.githubusercontent.com/69638013/203891275-2e8647fc-ab15-4cab-90b7-c4fa40109f1a.png)

Reply:
thanks i will do it right now